File: primes/Primes.java
public class Primes {
    public static void main(String[] args) {
        int numPrimes = 0;
        for (int i=2;i<250001;i++) {
            numPrimes += isPrime(i);
        }
        System.out.println(numPrimes);
    }
    public static int isPrime(int n) {
        for (int i=2;i<=n/2;i++)
            if((n%i) == 0)
                return 0;
        return 1;
    }
}

File: primes/primes.js
function isPrime(n) {
    for (let i=2; i<n/2;i++) {
        if(!(n%i)) {
            return 0
        }
    }
    return 1
}

let numPrimes=0
for (let i=2;i<250001;i++) {
    numPrimes+=isPrime(i)
}
console.log(numPrimes)